src/index.css

```diff
@@ -1,13 +1,5 @@
-body {
+*{
```

Comment:
Are you sure about giving `padding: 0` to every element on the application? Does that make sense?